README.md

```diff
@@ -89,17 +89,18 @@ pytest onnxscript
 ## Example
 
 ```python
-from onnx import TensorProto
-from onnx.helper import make_tensor
-from onnxscript import script, INT64, FLOAT
+from onnx
+from onnxscript import script
 
 # We use ONNX opset 15 to define the function below.
-from onnxscript import opset15 as op
+from onnxscript.onnx_opset import opset15 as op
```

Comment:
Why this change? I think it is better that we expose commonly used names (like opsets and types) in the top-level namespace. The internal organization is more of an implementation-issue that we can hide from external users.